AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -110,21 +105,6 @@ - (instancetype)init
   return [self initWithDataSource:fakeDataSource eventLog:eventLog];
 }
 
-- (void)setDelegate:(id<ASDataControllerDelegate>)delegate
-{
-  if (_delegate == delegate) {
-    return;
-  }
-  
-  _delegate = delegate;
-  
-  // Interrogate our delegate to understand its capabilities, optimizing away expensive respondsToSelector: calls later.
-  _delegateDidInsertNodes     = [_delegate respondsToSelector:@selector(dataController:didInsertNodes:atIndexPaths:withAnimationOptions:)];
-  _delegateDidDeleteNodes     = [_delegate respondsToSelector:@selector(dataController:didDeleteNodes:atIndexPaths:withAnimationOptions:)];
-  _delegateDidInsertSections  = [_delegate respondsToSelector:@selector(dataController:didInsertSections:atIndexSet:withAnimationOptions:)];
-  _delegateDidDeleteSections  = [_delegate respondsToSelector:@selector(dataController:didDeleteSectionsAtIndexSet:withAnimationOptions:)];
```

Comment:
I also deleted this in [#3017](https://github.com/facebook/AsyncDisplayKit/pull/3017/files#diff-c03b807fdd44839fd88691746d9dcc27L120). I will rebase with joy if necessary!